CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,8 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [VIF] To edit MAC address on a VIF, user have to be operator on this VIF and administrator and this network [#4700](https://github.com/vatesfr/xen-orchestra/issues/4700)([#5631](https://github.com/vatesfr/xen-orchestra/pull/5631))
```

Comment:
I don't think this was a bug, it was the intended behavior, and now, we're just changing it.